apps/prairielearn/src/ee/lib/context-parsers/documentation.ts

```diff
@@ -108,6 +116,39 @@ function cleanElementSections(elementSections: ElementSection[]) {
         }
         node.depth -= 1;
       }
+
+      if (node.type === 'table') {
+        const firstRow = node.children[0];
+        if (
+          firstRow.children.length === 4 &&
+          firstRow.children[0].children[0].value === 'Attribute' &&
+          firstRow.children[1].children[0].value === 'Type' &&
+          firstRow.children[2].children[0].value === 'Default' &&
+          firstRow.children[3].children[0].value === 'Description'
+        ) {
+          const optionalStatements: string[] = [];
+          const mandatoryStatements: string[] = [];
+          for (let row_idx = 1; row_idx < node.children.length; row_idx++) {
+            const row = node.children[row_idx];
+            const attribute = stringify([row.children[0]]).trimEnd();
+            const type = stringify([row.children[1]]).trimEnd();
+            const defaultVal = stringify([row.children[2]]).trimEnd();
+            const description = stringify([row.children[3]]).trimEnd();
+
+            if (defaultVal === '-') {
```

Comment:
Sometimes this is the `—` character (em dash) - you'll want to handle that too.

Reply:
done in c980442072ea4bf380653f218b44e399829697eb.